### DISCUSSION TITLE
Attempting to download zip crashes my firefox and the file deletes itself.

### DISCUSSION THREAD

**Discussion by IOnlyFetchBranches** [Sidekick-Poe/Sidekick#313] [Q&A] [answered]
As the title says, im trying to use this project but the zip file keeps deleting itself. I've got windows defender turned off, my malware bytes is also not detecting anything. Theres nothing in the history or quarantine. Whats going on here? Does firefox have some sort of in built virus detection, its not even saying anything it just says below 
![image](https://github.com/user-attachments/assets/bc7fb82f-2f8f-495c-8987-97fa50f89a8a)

**Comment by IOnlyFetchBranches** [comment, marked as answer]
Ugh it was nord, just found it in the logs.

**Reply by domialex** [reply]
Do share the error though, for posterity.

**Reply by IOnlyFetchBranches** [reply]
there was literally no error, it just straight up vanished into the ether. i finally opened it and you can see it under detections. why in gods name nord does this by default when i already have antivirus. idk.

**Reply by IOnlyFetchBranches** [reply]
ok so which one of the maintainers gets all my data hahaa!
